### DISCUSSION TITLE
Error when DBT_CLOUD_RUN_REASON contains a single quote

### DISCUSSION THREAD

**Discussion by ghost** [brooklyn-data/dbt_artifacts#322] [Q&A]
When a job is triggered through Airflow and DBT has Artefacts package it throws up compilation error

"DBT_CLOUD_RUN_REASON": "Triggered via Apache Airflow by task 'trigger_dbt_cloud_module' in the dv_batch DAG."

06:55:18  on-run-end failed, error:
06:55:18   001003 (42000): SQL compilation error:
06:55:18  syntax error line 52 at position 170 unexpected 'trigger_dbt_cloud_module'.

**Comment by jared-rimmer** [comment]
Hi @pramalingam2022. We have[ recently merged a PR](https://github.com/brooklyn-data/dbt_artifacts/pull/299/files) that resolves this issue and it will be available in the next release.

**Reply by ghost** [reply]
Thanks, any idea on timeline for upcoming release
